pkg/images/images.go

```diff
@@ -51,9 +51,11 @@ func GetImagePath(repo, version, image, component string, imageOverrides map[str
 }
 
 func GetPullPolicy(version, override string) v1.PullPolicy {
-	if len(override) > 0 {
-		tag := strings.Split(override, ":")[1]
-		return getPullPolicy(tag)
+	if len(override) > 0
```

Comment:
You don't need this check - `Split` handles empty string.

```suggestion
```

Reply:
I think we need this check,if the string is empty then we will unnecessary to split it

Reply:
As I mentioned, `Split` handles an empty string.

Reply:
Done